crypto/ecdsa/signer.go

```diff
@@ -31,7 +31,16 @@ func (x Signer) Sign(data []byte) ([]byte, error) {
 		return nil, err
 	}
 
-	return elliptic.Marshal(elliptic.P256(), r, s), nil
+	params := elliptic.P256().Params()
+	curveOrderByteSize := params.P.BitLen() / 8
+
+	buf := make([]byte, 1+curveOrderByteSize*2)
+	buf[0] = 4
+
+	_ = r.FillBytes(buf[1 : 1+curveOrderByteSize])
+	_ = s.FillBytes(buf[1+curveOrderByteSize:])
+
+	return buf, nil
```

Comment:
Shouldn't we move this to a separate function and add the same comments as in [neofs-crypto](https://github.com/nspcc-dev/neofs-crypto/blob/d29d7f9c3e8d6b6c14161af210e8aa1fa85bf36f/ecdsa.go#L42-L54)

Reply:
> move this to a separate function

that is used exactly in one place with the known points so i guess we do not need a separate function for that

> add the same comments

what exact comments do you mean?